packages/cardinal-integration/src/cardinal-three-d-secure-flow-v2.spec.ts

```diff
@@ -0,0 +1,208 @@
+import { Response } from '@bigcommerce/request-sender';
+import { merge } from 'lodash';
+
+import {
+    HostedForm,
+    OrderRequestBody,
+    PaymentIntegrationService,
+    PaymentMethod,
+    PaymentRequestOptions,
+    PaymentStrategy,
+    RequestError,
+} from '@bigcommerce/checkout-sdk/payment-integration-api';
+import {
+    getBillingAddress,
+    getCheckout,
+    getOrder,
+    getOrderRequestBody,
+    getPaymentMethod,
+    getResponse,
+    getShippingAddress,
+    PaymentIntegrationServiceMock,
+} from '@bigcommerce/checkout-sdk/payment-integrations-test-utils';
+
+import CardinalClient from './cardinal-client';
+import CardinalThreeDSecureFlowV2 from './cardinal-three-d-secure-flow-v2';
+
+describe('CardinalBarclaysThreeDSecureFlow', () => {
+    let cardinalClient: Pick<
+        CardinalClient,
+        'configure' | 'getThreeDSecureData' | 'load' | 'runBinProcess'
+    >;
+    let threeDSecureFlow: CardinalThreeDSecureFlowV2;
+    let paymentMethod: PaymentMethod;
+    let paymentIntegrationService: PaymentIntegrationService;
+
+    beforeEach(() => {
+        paymentMethod = getPaymentMethod();
+
+        cardinalClient = {
+            configure: jest.fn(() => Promise.resolve()),
+            getThreeDSecureData: jest.fn(() => Promise.resolve()),
+            load: jest.fn(() => Promise.resolve()),
+            runBinProcess: jest.fn(() => Promise.resolve()),
+        };
+
+        paymentIntegrationService = new PaymentIntegrationServiceMock();
+
+        threeDSecureFlow = new CardinalThreeDSecureFlowV2(
+            paymentIntegrationService,
+            // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
```

Comment:
Add this rule to `eslintrs.json` for test files

Reply:
Done